api.py

```diff
@@ -75,23 +64,23 @@ async def on_after_forgot_password(user: UserDB, token: str, request: Request):
             <html>
             <body>
             <h3>Recuperação de acesso</h3>
-            <p>Olá, {user.email},</p>
+            <p>Olá, {user.email}.</p>
             <p>Você esqueceu sua senha da API PGD.
-            Segue seu novo token de acesso: {token}
+            Segue seu novo token de acesso: <br/> {token}
             </p>
             </body>
             </html>
             """
     await send_email([user.email], subject, body)
 
 async def on_after_reset_password(user: UserDB, request: Request):
-    logging.info("User %s has reset their password.", user.id)
+    print("User %s has reset their password.", user.id)
```

Comment:
Does `print` work better than `logging.info` here? Why?

Reply:
Sorry...actually I was testing the print function but logging is better, and I forgot to revert this change.

Reply:
```suggestion
    logging.info("User %s has reset their password.", user.id)
```
Ok, so let's revert this line.